BitwardenShared/UI/Platform/Application/Appearance/Modifiers/ScrollViewModifier.swift

```diff
@@ -13,10 +13,13 @@ struct ScrollViewModifier: ViewModifier {
     /// The background color to apply to the scroll view.
     var backgroundColor: Color
 
+    /// Whether or not to show the scrolling indicators .
+    var showsIndicators = true
+
     // MARK: View
 
     func body(content: Content) -> some View {
-        ScrollView {
+        ScrollView(showsIndicators: showsIndicators) {
```

Comment:
@matt-livefront  this was also added in the other PR for the `LandingView`.  Which over goes in first ill fix any conflicts that arise.  I just didnt want to forge this on both.

Reply:
also not sure why this isn't being picked up by tests... do you see anything i did wrong herE?

Reply:
What's the reasoning behind hiding the scroll indicators?

Reply:
ah i should have explained that, my bad.  I ran this by Emily and was asked to hide them.